client/my-sites/plugins/plugin-details-CTA/staging-site-notice.jsx

```diff
@@ -0,0 +1,31 @@
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import Banner from 'calypso/components/banner';
+import productionSiteForWpcomStaging from 'calypso/state/selectors/get-production-site-for-wpcom-staging';
```

Comment:
Should we name this `getProductionSiteForWpcomStaging` for consistency with the other selectors?